### ISSUE TITLE
Add login

### ISSUE THREAD

**Issue by squallstar** [squallstar/gakusei#4]
TBD

**Comment by Gnarlywhale** [comment]
Is there a specific reason you wanted to integrate with twitter specifically as opposed to just a username+password?

Also, what were you planning on tracking with the user accounts (streak high-score, which phrases the user does the worst on, etc)?

**Comment by squallstar** [comment]
@Gnarlywhale no specific reason, but makes things much simpler as we don't have to care about forgot password, etc. Integrating a "Sign with XYZ" on Meteor is very simple and can be done in less than an hour.

We can add Google, Twitter, GitHub, Facebook, etc in just a couple of lines.

About tracking, yeah I was thinking something like that. Haven't come up with a plan yet but I guess I can start from basic things like those.

**Comment by Gnarlywhale** [comment]
@squallstar, cool I was just asking because I was going I was looking at integrating it, but twitter requires an access key and wasn't sure if you were planning on setting one up

**Comment by squallstar** [comment]
Yeah I'll set one up for learn.fragments.me though won't be shared with the repository. Keys will be stored in the settings file (as per meteor docs)
